doc/requirements/requirements.md

```diff
@@ -0,0 +1,306 @@
+PLT002 | Target Platforms.Operating Systems.MacOS | 1 | The Navigation System MUST support MacOS 10.13 (High Sierra) and MacOS 10.14 (Mohave)
+PLT003 | Target Platforms.Operating Systems.Windows | 1 | The Navigation System MUST support Windows 10 Professional
+PLT004 | Target Platforms.Operating Systems.Clear Linux | 1 | The Navigation System SHOULD support the latest version of Intel's Clear Linux distribution | What is the latest version number?
+PLT005 | Target Platforms.CPU.Word Size | 1 | The Navigation System SHALL support 64-bit processors | Don't assume a specific pointer size
+PLT006 | Target Platforms.Minimum Platform | 1 | *Should we specify a minimum target platform?* *Or, should this be expressed as minimum platform requirements?*
+
+## 2.4 Command Chain Modules
+
+This section lists the requirements for the core command chain modules in the Navigation System.
+
+### 2.4.1 Mission Planning
+
+A complete system, which incorporates the Navigation System, would also have a Mission Planning subsystem that generates a Mission Plan from input provided by the User. The Mission Plan is a sequence of primitive operations and any associated policy about how the plan should be carried out. The design and implementation of the subsystem that creates the Mission Plan is outside the scope of the Navigation System, but its output, the Mission Plan, has a direct relationship to the capabilities that the Navigation System must provide. 
+
+Id | Handle | Priority | Description | Notes
+-- | ------ | -------- | ----------- | -----
+MP001 | Mission Planning.Primitives | 1 | The Mission Plan MUST be able to express the plan as a sequence of the following Primitives (MP002 - MP010)
+MP002 | Mission Planning.Primitives.Navigate to Position | 1 | The Navigation System MUST be able to navigate the Robot from its current location to a specified destination pose.
+MP003 | Mission Planning.Primitives.Navigate to Area | 1 | The Navigation System MUST be able to navigate the Robot from its current location to a specified area. | *TODO: How to define "area"?*
+MP004 | Mission Planning.Primitives.Enqueue | 1 | The Navigation System MUST be able to navigate the Robot from its current location to a position behind another specified robot. 
+MP005 | Mission Planning.Primitives.Follow | 1 | The Navigation System MUST be able to direct the Robot to follow another specified robot. | This one doesn't have a completion state (reaching the goal), unless it specifies some more information such as "follow until destination reached"
+MP006 | Mission Planning.Primitives.Orbit | 1 | *TODO: How is this defined?* | Similar to Follow in that it doesn't complete (unless it specifies time)
+MP007 | Mission Planning.Primitives.Wait | 1 | The Navigation System MUST be able to idle at the current location until directed otherwise 
+MP008 | Mission Planning.Primitives.Park | 1 | *TODO: What does parking mean? Some low-power state?*
```

Comment:
Going to a low power state seems out of scope for the navstack.

Reply:
Do you think it would be automatically handled in the robot (if at all)?

Reply:
I agree, low power state seems out of scope.

Reply:
Does this mean dock to a charger? That maneuver could be a primitive, but then what happens next is platform dependent.